Tema 4/formulare modificat/index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Formular</title>
+        <link rel="stylesheet" href="style.css"
+    </head>
+    <body class="body">
+        <form class="form">
+            <fieldset style="display: inline-block;"><legend>Pizza Order</legend>
+            <label for="your-name">Your name:</label>
+            <input type="text" name="your-name" value="Malina" minlength="3" maxlength="10" required placeholder="Your name"/>
+            <br/>
+            <br/>
+            <label for="email"> Your e-mail:</label>
+            <input type="email" name="email" value="examplemail@gmail.com" required placeholder="Your e-mail"/>
+            <br/>
+            <br/>
+            <label for="topping">Topping:</label>
```

Comment:
Un label ii corespunde unui input.
Nu poti sa pui acelasi label la mai multe inputuri.
Aici corect ar fi fost ca Topping: sa fie pus in interiorul unui paragraf, sau alt element iar fiecare iput sa corespunda unui label
<p>Topping:</p>
<input type="radio" name="topping" id="supreme" value="Supreme"/>
<label for="supreme">Supreme</label>
<input type="radio" name="topping" id="vegetarian" value="Vegetarian"/>
<label for="vegetarian">Vegetarian</label>